docs/md/sink.md

```diff
@@ -0,0 +1,132 @@
+# Sink
+The workload `Sink` models the terminal stage of a processing pipeline. In a 
+typical workload configuration, a `Sink` can be used to write workflow outputs
+to a desired location in the cloud.
```

Comment:
Re: "in the cloud" - any chance of us expanding support to on-prem applications in the future?

Reply:
I certainly hope not.

Reply:
Sounds good, let's manifest the future that we want to see.